__tests__/app.test.js

```diff
@@ -97,6 +97,76 @@ describe('App Tests', () => {
                 })
             })
         })
+        describe('Added Feature: GET `/api/articles` test', () => {
+            test('topic query: filters the articles by topic value specified in the query.', () => {
+                return request(app)
+                .get('/api/articles?topic=mitch')
+                .then(({body: {articles}}) => {
+                    expect(articles.length).not.toBe(0);
+                    articles.forEach(article => {
+                        expect(article.topic).toBe('mitch');
+                    })
+                })
+            })
+            test('topic query: filters the articles by topic value specified in the query and is not case-sensitive.', () => {
+                return request(app)
+                .get('/api/articles?topic=MiTcH')
+                .then(({body: {articles}}) => {
+                    expect(articles.length).not.toBe(0);
+                    articles.forEach(article => {                        
+                        expect(article.topic).toMatch(/MiTcH/i);
```

Comment:
This assertion is being run on the response so you could assert `expect(article.topic).toBe("mitch")` again